specification/network/resource-manager/Microsoft.Network/stable/2019-04-01/checkPrivateLinkServiceVisibility.json

```diff
@@ -0,0 +1,83 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "NetworkManagementClient",
+    "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
+    "version": "2019-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow.",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/locations/{location}/checkPrivateLinkServiceVisibility": {
+      "post": {
+        "operationId": "CheckPrivateLinkServiceVisibility",
```

Comment:
Please check if this expected operation name (i.e. plain operation name instead of `Class_Operation` form).
Current value would place this operation directly under INetworkManagementClient like [CheckDnsNameAvailability](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/network/Microsoft.Azure.Management.Network/src/Generated/INetworkManagementClient.cs#L485)

Reply:
Thanks for the suggestion, I was thinking that the cmdlet that under different path from private link service,  should be a separate operation id. But since both you and Anton is suggested, I have already changed the operation id to merge them into private endpoint. This make code looks more neat. Thanks.